src/wp-includes/update.php

```diff
@@ -187,7 +187,8 @@ function wp_version_check( $extra_stats = array(), $force_check = false ) {
 	$doing_cron = wp_doing_cron();
 
 	$options = array(
-		'timeout'    => $doing_cron ? 30 : 3,
+		// Half a minute when doing cron tasks, 3 seconds otherwise.
+		'timeout'    => $doing_cron ? MINUTE_IN_SECONDS / 2 : 3,
```

Comment:
I see these constants best used for things like `2.5 * MINUTE_IN_SECONDS` rather than `150`, which provides context and saves the reader time in calculating the number of minutes.

However, I don't see how the proposed change makes things clearer. The need for a comment suggests that it doesn't. IMO, it's reasonable to expect a reader won't think `30 milliseconds : 3 milliseconds` or `30 minutes : 3 minutes`.